src/components/inputs/Selector.tsx

```diff
@@ -212,7 +219,11 @@ const AssigneeSelectorRenderer = ({ props, option }: { props: HTMLAttributes<HTM
           sx={{ width: '20px', height: '20px' }}
         />
         <Typography variant="sm" fontWeight={400}>
-          {(option as IAssigneeCombined)?.name || (option as IAssigneeCombined)?.givenName}
+          {truncateText(
+            (option as IAssigneeCombined)?.name ||
+              `${(option as IAssigneeCombined)?.givenName ?? ''}  ${(option as IAssigneeCombined)?.familyName ?? ''}`.trim(),
+            16,
```

Comment:
I think it's better to avoid magic numbers and create a constant for this since this number defines a part of UI.